src/frontends/common/src/frontend.cpp

```diff
@@ -108,3 +108,19 @@ std::string FrontEnd::get_name() const {
     }
     FRONTEND_RETURN_STATEMENT("Getting frontend name", m_actual->get_name();)
 }
+
+void FrontEnd::throw_path_error(const std::string& model_path) {
+    FRONT_END_GENERAL_CHECK(false, "Could not open the file: ", model_path);
```

Comment:
can we escape model_path with quotes? 
I saw when people passed extra spaces to model_path, so we can highlight it with such quotes